01_Core/week02/submissions/index.html

```diff
@@ -0,0 +1,109 @@
+        </div>
+        <div class="form">
+            <h2 id="you">Tell me about yourself!</h2>
+        <form action="post" method="GET">
+            <div>
+                <label for="name">First Name:</label>
+                <input type="text" id="first-name">
+            </div>
+            <div>
+                <label for="name">Last Name:</label>
+                <input type="text" id="last-name">
+            </div>
+            <div>
+                <label for="gender">Gender:</label><br>
+                <input type="radio" name="gender" value="female" checked>Female<br>
+                <input type="radio" name="gender" value="male">Male<br>
+                <input type="radio" name="gender" value="other">Other
+            </div>
+            <div>
+                <label for="sport">What's your favorite sport?</label><br>
+                <select name="sport">
+                    <option value="basketball">Basketball</option>
+                    <option value="baseball">Baseball</option>
+                    <option value="football">Football</option>
+                    <option value="golf">Golf</option>
+                    <option value="tennis">Tennis</option>
+                </select>
+            </div>
+            <div>
+                <fieldset>
+                    <legend>What do you like on your pizza?</legend>
+                    <input type="checkbox" name="topping" value="pepperoni" id="pepperoni">
+                    <label for="pepperoni">Pepperoni</label>
+                    <input type="checkbox" name="topping" value="pineapple" id="pineapple">
+                    <label for="pineapple">Pineapple</label>
+                    <input type="checkbox" name="topping" value="bacon" id="bacon">
+                    <label for="bacon">Bacon</label>
+                    <input type="checkbox" name="topping" value="ham" id="ham">
+                    <label for="ham">Ham</label>
+                    <input type="checkbox" name="topping" value="sausage" id="sausage">
+                    <label for="sausage">Sausage</label>
+                </fieldset>
+            </div>
+            <button type="submit">Save</button>
+        </form>
+    </div>
+        <div class="bottom">
+            <h2>Click <a href="another-page.html">here</a> for a list of people who have already completed this survey.</h2>
+        </div>
+    </body>
+    <footer>
+        <small>&copy; Copyright 2019</small>
+        <div id="social">
+            <a href="#">LinkedIn</a>
+            <a href="#">Twitter</a>
+            <a href="#">Facebook</a>
+        </div>
+        <div class="rainbow">
+                <h3>Oh, and here are the colors of the rainbow!</h3>
```

Comment:
Why did you do an h3 tag here? H-tags are hierarchical - h1s get child h2s. H2s get child h3s. So, when you use an h3, you are semantically indicating that it is a section of some h2 parent, which in this case I'm not sure you have.